src/renderer/src/i18n/fr-FR/artifacts.json

```
@@ -36,6 +36,12 @@
   "copyFailedDesc": "Impossible de copier le contenu dans le presse-papiers",
   "copyAsImage": "Copier en tant qu'image",
   "copyImageSuccessDesc": "Image copiée dans le presse-papiers",
+  "copyImageFailedDesc": "Impossible de copier des images dans le presse-papiers",
```

Biome (2.1.2)

[error] The key copyImageFailedDesc was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)